<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomCreation</class>
 <widget class="QDialog" name="RoomCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>173</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>175</width>
    <height>173</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>175</width>
    <height>173</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Room Creation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #E3EAFF;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="widthInputFieldHint">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="widthInputField">
         <property name="styleSheet">
          <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lengthtInputFieldHint">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lengthInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="heightInputFieldHint_2">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="heightInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="confirmCreationButton">
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
background: white;
border-radius: 5px; 
height: 30px;
}
QPushButton:hover
{
background: rgb(244, 244, 244);
}


</string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
background: white;
border-radius: 5px; 
height: 30px;
}
QPushButton:hover
{
background: rgb(244, 244, 244);
}


</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
